roles/veyepar-encoder/tasks/main.yml

```diff
@@ -3,36 +3,65 @@
   apt:
     name: "{{ item }}"
   with_items:
+  - python-psycopg2
   - libpq5
   - imagemagick
   - python3-gi
   - gstreamer1.0-tools
   - gir1.2-gstreamer-1.0
   - gir1.2-gst-plugins-base-1.0
   - gstreamer1.0-plugins-good
+  - gstreamer1.0-plugins-ugly
```

Comment:
Hmm..    I am going to guess either tsraw (read start/duration from headers) or gslevels (make the audio level visualizations (which we didn't do at LCA, which would explain why we didn't need it.  I think...